### PR TITLE
feat: integrate with chronicle in the order handler (initial test)

### PR DESCRIPTION
**Pull request by petrovska-petro**
run:

```
forget test -vvvv
```

NOTE: net is vvvv slow so it may take sometime

Screenshot 2023-11-18 at 12 52 17

### CODE REVIEW

File: test/TestOracleFeed.t.sol [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+contract TestOracleFeed is BaseFixture {
+    function setUp() public override {
+        BaseFixture.setUp();
+    }
+
+    function testOracleFeedWhitelisting() public {
+        // ensure sc is WL
+        bool isTooled = ORACLE_CHRONICLE.tolled(address(orderHandler));
+        assertTrue(isTooled);
```
Comment (gosuto-inzasheru) [inline test/TestOracleFeed.t.sol]:
```suggestion
        bool isTolled = ORACLE_CHRONICLE.tolled(address(orderHandler));
        assertTrue(isTolled);
```

File: src/OrderHandler.sol [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    // ref: https://chroniclelabs.org/dashboard/oracle/ETH/USD?blockchain=GNO&txn=0x553acf4265afea825633c10a3252e7ab532c655745a11c47d4cdff8d3714b725&contract=0x5e16ca75000fb2b9d7b1184fa24ff5d938a345ef
+    IChronicleOracle constant ORACLE_CHRONICLE = IChronicleOracle(0x5E16CA75000fb2B9d7B1184Fa24fF5D938a345Ef);
+
+    ComposableCoW public immutable composableCow;
+
+    constructor(ComposableCoW _composableCow) {
+        composableCow = _composableCow;
+    }
+
+    function kissMe() external {
```
Comment (gosuto-inzasheru) [inline src/OrderHandler.sol]: is this allowed on gnosis mainnet?

Comment (petrovska-petro) [reply src/OrderHandler.sol]: afaik you can do the WL in 2 ways one is thru the [self-kiss SC](https://gnosisscan.io/address/0x0Dcc19657007713483A5cA76e6A7bbe5f56EA37d#code#F1#L52) which basically calls the `kiss` method of the oracle itself as a helper or directly go into the [oracle](https://gnosisscan.io/address/0x5e16ca75000fb2b9d7b1184fa24ff5d938a345ef#code#F4#L70) with an auth addy (which i guess is their devs) and get auth by impersonating them in the meantime

Comment (petrovska-petro) [reply src/OrderHandler.sol]: resolved in 91def8a15a3aa4fa451cf2dea4b5ed606e8f7959

**Pull request merged**